TeamProject/GlobalWarmingGame/SoundFactory.cs

```diff
@@ -10,65 +10,49 @@ namespace GlobalWarmingGame
 {
     public static class SoundFactory
     {
-        public static Dictionary<string, SoundEffect> sounds;
-        public static Dictionary<string, Song> songs;
+        public static Dictionary<Sound, SoundEffect> sounds;
+        public static Dictionary<PFSong, Song> songs;
         public static Song song; 
         public static void Loadsounds(ContentManager content)
         {
-            sounds = new Dictionary<string, SoundEffect>();
-            songs = new Dictionary<string, Song>();
-            Song song = null;
-            songs.Add("menu",content.Load<Song>("sound/songs/menu"));
-            songs.Add("main",content.Load<Song>("sound/songs/ColdAtmosphericMusic"));
-            songs.Add("enemy_zone", content.Load<Song>("sound/songs/enemy_zone"));
-            sounds.Add("wood_chop", content.Load<SoundEffect>(@"sound/sounds/wood_chop"));
-            sounds.Add("rabbit_death", content.Load<SoundEffect>(@"sound/sounds/rabbit_death"));
-            sounds.Add("stone_pickup", content.Load<SoundEffect>(@"sound/sounds/stone_pickup"));
-        }
-        static public void PlayGameSoundtrack()
-        {
-            song = songs["main"];
-            MediaPlayer.IsRepeating = true;
-            MediaPlayer.Play(song);
-            MediaPlayer.MediaStateChanged += MediaPlayer_MediaStateChanged;
+            songs = new Dictionary<PFSong, Song>
+            {
+                { PFSong.Menu,      content.Load<Song>(@"sound/songs/menu") },
+                { PFSong.Game,      content.Load<Song>(@"sound/songs/ColdAtmosphericMusic") },
+                { PFSong.EnemyZone, content.Load<Song>(@"sound/songs/enemy_zone") }
+            };
+            sounds = new Dictionary<Sound, SoundEffect>
+            {
+                { Sound.WoodChop,       content.Load<SoundEffect>(@"sound/sounds/wood_chop") },
+                { Sound.RabbitDeath,    content.Load<SoundEffect>(@"sound/sounds/rabbit_death") },
+                { Sound.StonePickup,    content.Load<SoundEffect>(@"sound/sounds/stone_pickup") }
+            };
         }
-        static public void PlayGameMenuSong()
+        static public void PlaySong(PFSong s, bool repeate = true)
```

Comment:
We are going to have many more songs in the future, and having seperate methods for them could get a little messy, So an enum can be used instead.